product_product_template_link/views/product_product_views.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">product.product.normal.form</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_normal_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//button[hasclass('oe_stat_button')]" position="before">
```

Comment:
Currently this xpath expression targets two different elements in the inherited templates. I would suggest to be more explicit by replacing the xpath with `<button name="open_pricelist_rules" position="before">....</button>`.

Reply:
@sanderlienaerts you are right, thank you!